dll/win32/shell32/lang/en-US.rc

```diff
@@ -110,8 +110,8 @@ BEGIN
     EDITTEXT IDC_BROWSE_FOR_FOLDER_FOLDER_TEXT, 12, 38, 194, 14, ES_AUTOHSCROLL | WS_GROUP
     CONTROL "", IDC_BROWSE_FOR_FOLDER_TREEVIEW, "SysTreeView32", TVS_SHOWSELALWAYS | TVS_EDITLABELS | TVS_LINESATROOT | TVS_HASLINES | TVS_HASBUTTONS | WS_BORDER | WS_TABSTOP, 12, 58, 194, 105
     PUSHBUTTON "&Make New Folder", IDC_BROWSE_FOR_FOLDER_NEW_FOLDER, 12, 174, 77, 14, WS_GROUP | WS_TABSTOP
-    DEFPUSHBUTTON "OK", IDOK, 102, 174, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
-    PUSHBUTTON "Cancel", IDCANCEL, 156, 174, 50, 14, WS_GROUP | WS_TABSTOP
+    DEFPUSHBUTTON "OK", IDOK, 102, 174, 50, 14, BS_DEFPUSHBUTTON | WS_TABSTOP
+    PUSHBUTTON "Cancel", IDCANCEL, 156, 174, 50, 14, WS_TABSTOP
```

Comment:
Similarly, the `PUSHBUTTON` already contains `WS_TABSTOP`, so that one too can be removed
https://learn.microsoft.com/en-us/windows/win32/menurc/pushbutton-control

Reply:
> Similarly, the `PUSHBUTTON` already contains `WS_TABSTOP`, so that one too can be removed https://learn.microsoft.com/en-us/windows/win32/menurc/pushbutton-control

So basically those styles are unnecesary and the buttons should look like this?
```
DEFPUSHBUTTON "OK", IDOK, 102, 174, 50, 14
PUSHBUTTON "Cancel", IDCANCEL, 156, 174, 50, 14
```

Reply:
The thing is that we most likely do not want IDOK to have a WS_TABSTOP (which it would have by default as a DEFPUSHBUTTON).
Since there is a   PUSHBUTTON "&Make New Folder" at the line before it, and we want to be able to navigate between these 3 buttons with the arrow keys. They are in the same group.

I would therefore recommed to use
```
DEFPUSHBUTTON "OK", IDOK, 102, 174, 50, 14, BS_DEFPUSHBUTTON
PUSHBUTTON "Cancel", IDCANCEL, 156, 174, 50, 14
```
instead. Or (if that wouldn't work yet) we could use:
```
DEFPUSHBUTTON "OK", IDOK, 102, 174, 50, 14, BS_DEFPUSHBUTTON | NOT WS_TABSTOP
PUSHBUTTON "Cancel", IDCANCEL, 156, 174, 50, 14
```

That should behave better than:
```
PUSHBUTTON "&Make New Folder", IDC_BROWSE_FOR_FOLDER_NEW_FOLDER, 12, 174, 77, 14, WS_GROUP | WS_TABSTOP
DEFPUSHBUTTON "OK", IDOK, 102, 174, 50, 14
PUSHBUTTON "Cancel", IDCANCEL, 156, 174, 50, 14
```

But like I said: That was already broken before your changes.

Reply:
Don't strip any WS_TABSTOP here. **The PR is correct like it currently is.**
I tested on WinXP the browseForFolderDialog by starting rapps and navigating into its properties.
**All 3 buttons seem to have a WS_TABSTOP.**
I can tabtabtab and it will select each of it then.
But still I can use the arrow-keys to navigate between them which means only the first one has a WS_GROUP style.

**The PR can be merged as is and should not be changed.**

Reply:
No problem. Those styles have been cleaned and the menu works fine :)

Reply:
@JoachimHenze
>  I tested on WinXP the browseForFolderDialog by starting rapps and navigating into its properties.
All 3 buttons seem to have a WS_TABSTOP.

`PUSHBUTTON` _**already**_ contains `WS_TABSTOP`. You see it in the resource because you've also `CONTROL xxxx "Button", ..., BS_PUSHBUTTON | WS_TABSTOP`, which is the resource expansion of `PUSHBUTTON`.

Reply:
Yes, I do understand that.